charmcraft.yaml

```
@@ -175,3 +175,8 @@ config:
       type: string
       default: info
       description: Log level for the UPF. One of `debug`, `info`, `warn`, `error`, `fatal`, `panic`.
+    bess-http:
```

```suggestion
    enable-bess-http:
```
can we rename the config parameter to something like this ? it would be more evident that it's a boolean parameter.